templates/gameday2.html

```diff
@@ -50,7 +50,7 @@
       return ((parseInt(t[1]) || 0) + 1) + ':00';
     })(tos.split(':').reverse());
     // Collect and send the time to Google Analytics
-    window.pageTracker ? pageTracker._trackEvent('gameday_time', 'Log', tos) : _gaq.push(['_trackEvent', 'gameday_time', 'Log', tos]);
+    ga('send', 'event', 'gameday_time', 'gameday_time', 'Log', tos);
```

Comment:
cc @fangeugene Can you double check this looks right? I don't want to break GameDay tracking